en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+# Safety Configuration (Failsafes) 
+
+PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+
+The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+
+## Failsafe Actions {#failsafe_actions}
```

Comment:
The failsafe actions are often things like Return to home, Land, Loiter.
1. My assumption is that these are **exactly** like the modes, Return, Land, Hold - except presumably the system already knows that they are handled in a failsafe, and can offer different actions if a further failsafe occurs:
   * Is this correct?
   * If so, should we not mark up the parameters to use the new names for the modes - Return, Land, Hold etc? (or Return mode, Land mode, Hold mode). This is especially true as return to home is no longer going to be necessarily the destination.

Reply:
Great. Done in https://github.com/PX4/Firmware/pull/9429